### ISSUE TITLE
'make dev' giving error when not on the latest version of npm 

### ISSUE THREAD

**Issue by andrewhuanggg** [openfoodfacts/openfoodfacts-server#10176]
### What
- When I executed the 'make dev' command, I ran into errors with the final error message being: "failed to solve: process "/bin/sh -c npm install". 

### Solution
- It seems like the error was due to the fact that my npm version was 10.5.0, but the latest version is 10.6.0. So if anyone else is running into this error, all I had to do is `npm install -g npm@10.6.0` in order to fix it.

**Comment by hangy** [comment]
To be honest, I don't quite understand. When running `make dev`, `node` and `npm` are only run in a `docker` container. In fact, my WSL2 doesn't have either `npm`, nor `node` installed, and I can use `make dev` without any issues.

Do you happen to have any more detailed logs of when it failed?

**Comment by JagjeevanAK** [comment]
I agree with @hangy since npm is being used inside the Docker container, the remote npm version or any related errors are not relevant here.

Hey @andrewhuanggg can you retry it as make file has been updated by @rpowis and let us know that does this issue still persist.

**Comment by VaiTon** [comment]
Closing as can't reproduce. Feel free to reopen if anything changes!
